packages/console/src/components/CardTitle/index.tsx

```diff
@@ -9,32 +9,18 @@ type Props = {
   subtitle?: AdminConsoleKey;
 };
 
-type RawProps = {
-  title: ReactNode;
-  subtitle?: ReactNode;
-};
-
-/**
- * Do not use this component directly, unless you do not want to use translation.
- */
-export const RawCardTitle = ({ title, subtitle }: RawProps) => (
-  <div>
-    <div className={styles.title}>{title}</div>
-    {subtitle && <div className={styles.subtitle}>{subtitle}</div>}
-  </div>
-);
-
 /**
  * Always use this component to render CardTitle, with built-in i18n support.
  */
 const CardTitle = ({ title, subtitle }: Props) => {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const props: RawProps = {
-    title: t(title),
-    subtitle: subtitle ? t(subtitle) : undefined,
-  };
 
-  return <RawCardTitle {...props} />;
+  return (
+    <div>
+      <div className={styles.title}>{t(title)}</div>
+      {subtitle && <div className={styles.subtitle}>{t(subtitle)}</div>}
+    </div>
+  );
 };
```

Comment:
just want to double-check, by design it only accepts i18nkey?

Reply:
Yes, per the discussion yesterday, I removed the `Raw` component, and removed all the references to the Raw component.
Will implement the `DangerousRaw` approach when needed in the future.